docs/src/main/sphinx/object-storage/file-system-s3.md

```diff
@@ -68,7 +68,16 @@ support:
 * - `s3.http-proxy`
   - URL of a HTTP proxy server to use for connecting to S3.
 * - `s3.http-proxy.secure`
-  - Set to `true` to enable HTTPS for the proxy server..
+  - Set to `true` to enable HTTPS for the proxy server.
```

Comment:
Please revert unrelated changes.

Reply:
Hi @ebyhr,
Sorry, it's a result of syncing up my branch with the master from upstream.
Please advise on how to proceed. I am actually considering opening a new PR as my git tree is messy.
Thanks.